addons/gdUnit4/test/core/command/GdUnitCommandHandlerTest.gd

```diff
@@ -51,19 +51,53 @@ func _test__check_test_run_stopped_manually() -> void:
 	verify(inspector, 1).cmd_stop(inspector._client_id)
 
 
-func test_scan_test_directorys() -> void:
-	assert_array(GdUnitCommandHandler.scan_test_directorys("res://", "test", [])).contains_exactly([
+func test_scan_test_directories() -> void:
+	assert_array(GdUnitCommandHandler.scan_test_directories("res://", "test", [])).contains_exactly([
 		"res://addons/gdUnit4/test"
 	])
 	# for root folders
-	assert_array(GdUnitCommandHandler.scan_test_directorys("res://", "", [])).contains_exactly([
+	assert_array(GdUnitCommandHandler.scan_test_directories("res://", "", [])).contains_exactly([
 		"res://addons", "res://assets", "res://gdUnit3-examples"
 	])
-	assert_array(GdUnitCommandHandler.scan_test_directorys("res://", "/", [])).contains_exactly([
+	assert_array(GdUnitCommandHandler.scan_test_directories("res://", "/", [])).contains_exactly([
 		"res://addons", "res://assets", "res://gdUnit3-examples"
 	])
-	assert_array(GdUnitCommandHandler.scan_test_directorys("res://", "res://", [])).contains_exactly([
+	assert_array(GdUnitCommandHandler.scan_test_directories("res://", "res://", [])).contains_exactly([
 		"res://addons", "res://assets", "res://gdUnit3-examples"
 	])
 	# a test folder not exists
-	assert_array(GdUnitCommandHandler.scan_test_directorys("res://", "notest", [])).is_empty()
+	assert_array(GdUnitCommandHandler.scan_test_directories("res://", "notest", [])).is_empty()
+
+
+func test_scan_all_test_directories() -> void:
+	var original_test_root_folder := GdUnitSettings.test_root_folder()
+	var test_root_folder_property := GdUnitSettings.get_property(GdUnitSettings.TEST_LOOKUP_FOLDER)
+
+	# Test when test_root_folder is blank
+	test_root_folder_property.set_value("")
+	GdUnitSettings.update_property(test_root_folder_property)
+	assert_array(GdUnitCommandHandler.scan_all_test_directories()).contains_exactly(["res://"])
```

Comment:
the tests are broken
To avoid using `GdUnitSettings` do use an argument on `scan_all_test_directories()` to fit the test_root_folder